app/serializers/activitypub/activity_serializer.rb

```diff
@@ -12,11 +12,55 @@ def self.serializer_for(model, options)
     end
   end
 
+  class SynchronizationItemSerializer < ActivityPub::Serializer
+    include RoutingHelper
+
+    context :security
+    context_extensions :collection_synchronization
+
+    class DigestSerializer < ActivityPub::Serializer
+      attributes :type, :digest_algorithm, :digest_value
+
+      def type
+        'Digest'
+      end
+
+      def digest_algorithm
+        'http://www.w3.org/2001/04/xmlenc#sha256'
+      end
+
+      def digest_value
+        object
+      end
+    end
+
+    attributes :type, :partial_collection, :domain
+    has_one :virtual_object, key: :object
+    has_one :digest, serializer: DigestSerializer
+
+    def type
+      'SynchronizationItem'
+    end
+
+    def virtual_object
+      account_followers_url(object.account)
+    end
+
+    def partial_collection
+      account_followers_synchronization_url(object.account)
+    end
+  end
+
   attributes :id, :type, :actor, :published, :to, :cc
+  has_many :collection_synchronization, serializer: SynchronizationItemSerializer if: :collection_synchronization?
```

Comment:
need a comma here
```suggestion
  has_many :collection_synchronization, serializer: SynchronizationItemSerializer, if: :collection_synchronization?
```

Reply:
yup, shuffling things around, didn't catch that!